torch/_tensor_docs.py

```diff
@@ -5467,15 +5517,43 @@ def callable(a, b) -> number
     >>> sparse._nnz()
     25
 
+    >>> dense = torch.zeros(3, 3, 1, 1)
+    >>> dense[0, 0] = dense[1, 2] = dense[2, 1] = 0
```

Comment:
Should this be `= 1`?

Reply:
It should, strange that output is correct i.e. as that `= 1` was there.  In any case, an update pushed.